changelog/fix_false_negatives_in_style_safenavigation_20250530140637.md

```diff
@@ -0,0 +1 @@
+* [#14216](https://github.com/rubocop/rubocop/pull/14216): False negatives in `Style/SafeNavigation`. ([@steiley][])
```

Comment:
Can you update the link to point to the original issue, and also include a bit more detail about the fix? e.g.,

```suggestion
* [#14205](https://github.com/rubocop/rubocop/issues/14205): False negatives in `Style/SafeNavigation` when a ternary expression is used in a method argument. ([@steiley][])
```

Reply:
Thanks for good advises.
I updated this.